### PR TITLE
Add 'metastores get-assignment' CLI for UC

### PR DESCRIPTION
**Pull request by adamcain-db**
* Adds `unity-catalog metastores get-assignment` (a.k.a. `unity-catalog get-metastore-assignment`) for exercising new UC `getCurrentMetastoreAssignment` API endpoint

### CODE REVIEW

File: databricks_cli/unity_catalog/metastore_cli.py
Comment (wchau) [inline databricks_cli/unity_catalog/metastore_cli.py:209]: Do we need this? Based on the comment, these are "deprecated" commands

Comment (adamcain-db) [reply databricks_cli/unity_catalog/metastore_cli.py:209]: No, I just kinda liked the `get-metastore-assignment` version.  Can remove it, if you prefer.

**Review by wchau — APPROVED** [review]

**Comment by adamcain-db** [comment]
I'll wait for @pietern to have a chance to review before I merge this.

**Review by pietern — APPROVED** [review]
LGTM

@adamcain-db Can you add one or two UC tests to set a precedent? Would be good to ensure new functionality gets tested. For this one it would be good to have a mocked response payload and expectations on the output. It looks like the current approach just dumps the JSON returned by the server, but it might be more in line with expectations to only print the metastore ID.

**Comment by pietern** [comment]
@adamcain-db Please enable commit signing and repush so we can merge.

See https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

**Comment by adamcain-db** [comment]
@pietern:

> It looks like the current approach just dumps the JSON returned by the server, but it might be more in line with expectations to only print the metastore ID.

I think we should keep dumping the full JSON (as done with the other `unity-catalog` commands).  Turns out, the `default_catalog_name` part of the assignment is of major interest to some users (e.g., the Partner Connect folks).

I'll add unit tests in a separate PR.

**Comment by pietern** [comment]
Understood, thanks for clarifying.

**Pull request merged**
